### ISSUE TITLE
How to calculate depth in Meters from the depth map in numpy

### ISSUE THREAD

**Issue by Luqman-Ansari** [prs-eth/Marigold#123]
Hello, I  have been working on a project of object detection and depth estimation. This model aligns to the my project requirements. I am using your pretrained model to test model accuracy on my own dataset. **Can you please provide formula or method on how to calculate exact depth in meters from the depth map generated using your model.** I cannot find any information in the research paper too related to this.

Please provide sufficient information so that I can contine to use this model for my project. Thanks!

**Comment by Luqman-Ansari** [comment]
Have you implemented any techniques or formula to convert the generated depth map into exact distance values ?

**Comment by markkua** [comment]
Hi, the prediction is up to unknown scale and shift factors. If you need metric depth (in meter), you can use other information as guidance to align, e.g. lidar point measurement, or taking other metric depth prediction and align our prediction to it.

**Comment by booker-max** [comment]
> Hi, the prediction is up to unknown scale and shift factors. If you need metric depth (in meter), you can use other information as guidance to align, e.g. lidar point measurement, or taking other metric depth prediction and align our prediction to it.

Actually, I've found that even after aligning the data using this method, the results are still not ideal. The estimated affine-invariant depth is not consistently accurate. I've tested this approach on several autonomous driving datasets, and even after applying proper scale and shift adjustments, the geometric results are still not correct. I believe that if the method could be fine-tuned for specific datasets, the metric performance could be significantly improved. Therefore, I'd like to ask how I can fine-tune this method to achieve better metric results.
